[2025-09-24T19:44:19.407+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_etl_neotass.transform manual__2025-09-24T19:43:56.392953+00:00 [queued]>
[2025-09-24T19:44:19.535+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_etl_neotass.transform manual__2025-09-24T19:43:56.392953+00:00 [queued]>
[2025-09-24T19:44:19.537+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-09-24T19:44:19.730+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform> on 2025-09-24 19:43:56.392953+00:00
[2025-09-24T19:44:19.851+0000] {standard_task_runner.py:60} INFO - Started process 2937 to run task
[2025-09-24T19:44:19.911+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_etl_neotass', 'transform', 'manual__2025-09-24T19:43:56.392953+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_neotass.py', '--cfg-path', '/tmp/tmpt0vss0or']
[2025-09-24T19:44:19.932+0000] {standard_task_runner.py:88} INFO - Job 23: Subtask transform
[2025-09-24T19:44:20.883+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_etl_neotass.transform manual__2025-09-24T19:43:56.392953+00:00 [running]> on host ad6d7be6e1d5
[2025-09-24T19:44:22.592+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_etl_neotass' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T19:43:56.392953+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-24T19:43:56.392953+00:00'
[2025-09-24T19:44:28.093+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/dag_etl_neotass.py:82 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-09-24T19:44:28.102+0000] {logging_mixin.py:188} INFO - Directory already exists
[2025-09-24T19:44:32.369+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T19:44:32.437+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_etl_neotass, task_id=transform, execution_date=20250924T194356, start_date=20250924T194419, end_date=20250924T194432
[2025-09-24T19:44:32.753+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T19:44:33.232+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
